integration_tests/integration_tests/test_table_monitoring_query.py

```diff
@@ -0,0 +1,138 @@
+from datetime import datetime
+from dbt_osmosis.core.osmosis import DbtProject
```

Comment:
Why use `dbt-osmosis` over `dbt-core`?
Does it have a compatibility layer for different versions of dbt?

Reply:
So yeah I think so - they created wrappers on top of the dbt-core objects for this purpose.
In general it looks like a relatively well maintained repo and the API they provided is nice and relatively simple (but not without caveats).

I admit though that I didn't try to use dbt-core directly so we can try that as well :)